lib/responses.test.js

```diff
@@ -410,4 +410,37 @@ describe(_.startCase(filename), function () {
       fn()(req, res);
     });
   });
+
+  describe('checkArchivedToPublish', function () {
+    const fn = lib[this.title];
+
+    it('throws an error if checkProps returns true', function () {
+      let req = {
+        body: { archived: true },
+        uri: 'domain.com/_pages/id'
+      };
+      const res = createMockRes();
+
+      db.getMeta.returns(Promise.resolve({published: true}));
```

Comment:
Please mind the spacing
```suggestion
      db.getMeta.returns(Promise.resolve({ published: true }));
```